src/main/webapp/js/datahandlers/collegepagedatahandler.js

```diff
@@ -19,9 +19,7 @@
 
 goog.module('datahandlers.collegepage');
 
-const {CollegeQueryBuilder, ID, NAME, ACCEPTANCE_RATE, ACT_SCORE, ANNUAL_COST, 
-    FIRST_NET_COST, SECOND_NET_COST, THIRD_NET_COST, FOURTH_NET_COST, 
-    FIFTH_NET_COST, MEDIAN_DEBT} = goog.require('datahandlers.collegequerybuilder');
+const {ACCEPTANCE_RATE, ACT_SCORE, CollegeQueryBuilder, FIFTH_NET_COST, FIRST_NET_COST, FOURTH_NET_COST, MEDIAN_DEBT, NAME, SECOND_NET_COST, THIRD_NET_COST} = goog.require('datahandlers.collegequerybuilder');
```

Comment:
Needs to be line-wrapped, it's way more than 80 chars.

Reply:
goog.require statements are excluded from the 80 columns limit. (https://google.github.io/styleguide/jsguide.html#formatting-column-limit)